tools/smartos-release

```diff
@@ -78,31 +82,40 @@ set -o errexit
 # Note that ${BRANCH} appears only once here as we assume that for release
 # builds, all project branches and smartos-live itself use the same branch
 # name. See $(PUB_BRANCH_DESC) in the top-level Makefile for more details.
-SOURCE=/${MANTA_USER}/public/builds/platform/${BRANCH}-${TIMESTAMP}/platform/
+SOURCE=/${MANTA_USER}/public/builds/platform/${BRANCH}-${TIMESTAMP}/platform
 SMARTOS=/${MANTA_USER}/public/SmartOS
 DESTINATION=${SMARTOS}/${TIMESTAMP}
 
 start_time=$(date +%s)
-echo "Creating release snaplinks under ${DESTINATION}"
+echo "Creating release objects under ${DESTINATION}"
 
 mmkdir -v -p ${DESTINATION}
-mfind ${SOURCE} -t o | while read OBJECT; do
-    mln ${OBJECT} ${DESTINATION}/$(basename ${OBJECT})
-done
-
-echo "Updating top level ${SMARTOS} snaplinks"
-mln ${SOURCE}/platform-${BRANCH}-${TIMESTAMP}.tgz ${SMARTOS}/platform-latest.tgz
-mln ${SOURCE}/smartos-${TIMESTAMP}.iso ${SMARTOS}/smartos-latest.iso
-mln ${SOURCE}/smartos-${TIMESTAMP}-USB.img.gz ${SMARTOS}/smartos-latest-USB.img.gz
-mln ${SOURCE}/smartos-${TIMESTAMP}.vmwarevm.tar.gz ${SMARTOS}/smartos-latest.vmwarevm.tar.gz
+# Add an object pointing to the actual platform Manta directory, along with
+# a html version, for convenience.
+echo ${SOURCE} | mput ${DESTINATION}/platform-link
```

Comment:
set content-type like the 'latest' file?

Reply:
good point.